<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Responsable</class>
 <widget class="QMainWindow" name="Responsable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1111111</width>
     <height>1111111</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/Icons/Icons/add_users.png);
padding : 1px  1px;
border:none;
}
QPushButton:hover {
  background: #3cb0fd;
  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
  text-decoration: none;
Border-Radius:15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/add-users.png</normaloff>:/Icons/add-users.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="Conge">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>711</width>
      <height>620</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>620</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
  border: 1px solid lightgray;
  top:1px; 
  background: rgb(245, 245, 245);; 
} 

QTabBar::tab {
  background: rgb(230, 230, 230); 
  border: 1px solid lightgray; 
  padding: 10px;
} 

QTabBar::tab:selected { 
  background: rgb(188, 215, 220); 
  margin-bottom: -1px; 
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Conge">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>801</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Demande de Conge</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>20</y>
        <width>691</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horiz_ayoub"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tous les demandes</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>55</y>
        <width>701</width>
        <height>511</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>N° de Matricule</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prénom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Congé demandé</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date de début</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre des jours</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date de Retour</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>0</y>
        <width>751</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>5</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Icons/Icons/refresh-removebg-preview.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>   Actualiser</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: rgb(249, 105, 98 );
border : none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(190, 254, 179);
border : none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(252, 237, 191);
border : none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>11</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:a 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Congé refusé </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>11</y>
         <width>90</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:a 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Congé accepté </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>11</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:a 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Congé en attente</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>5</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Icons/Icons/refresh-removebg-preview.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string> Notification</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(183, 254, 213 );
border : none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:a 9pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Accepté par RH</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;MS Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>Aucune demande ...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Compte</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>711</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/Icons/Icons/icon-identity-card.png);
border : none;
padding : -90px 60px -50px;
background-color: qlineargradient(spread:pad, x1:0.483364, y1:1, x2:0.483409, y2:0.074, stop:0 rgba(220, 210, 249, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Barlow</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
    font-family: Barlow, sans-serif;
    font-size:  15px;
    color: white;
    text-transform: none;
    font-style: bold;
    text-decoration: none;
    line-height: 1em;
    letter-spacing: 3px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Barlow</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
    font-family: Barlow, sans-serif;
    font-size:  12px;
    color: white;
    text-transform: none;
    font-style: bold italic;
    text-decoration: none;
    line-height: 1em;
    letter-spacing: 3px;
}</string>
      </property>
      <property name="text">
       <string>soukaina</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>350</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Barlow</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
    font-family: Barlow, sans-serif;
    font-size:  12px;
    color: white;
    text-transform: none;
    font-style: bold;
    text-decoration: none;
    line-height: 1em;
    letter-spacing: 3px;
}</string>
      </property>
      <property name="text">
       <string>Ayoub</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>500</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>     Log Out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>111</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_6{
image: url(:/Icons/Icons/logo.png);
padding : 5px;
	background-color: qlineargradient(spread:pad, x1:0.931818, y1:0.244, x2:0.955, y2:1, stop:0 rgba(190, 181, 244, 255), stop:1 rgba(255, 255, 255, 255));
border:none;
border-radius:3px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icons/add-users.png</normaloff>:/Icons/add-users.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Barlow</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
    font-family: Barlow, sans-serif;
    font-size:  12px;
    color: white;
    text-transform: none;
    font-style: bold;
    text-decoration: none;
    line-height: 1em;
    letter-spacing: 3px;
}</string>
      </property>
      <property name="text">
       <string>Matricule : </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/Icons/Icons/account-removebg-preview.png);
padding : -10px;
border:none;
}
QPushButton:hover {
  background: #3cb0fd;
  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
  text-decoration: none;
Border-Radius:15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/add-users.png</normaloff>:/Icons/add-users.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/Icons/Icons/list-removebg-preview.png);
padding : 4px 3px;
border:none;
}
QPushButton:hover {
  background: #3cb0fd;
  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
  text-decoration: none;
Border-Radius:15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/Icons/add-users.png</normaloff>:/Icons/add-users.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>7</pointsize>
      <italic>false</italic>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>AEROAUTO MAROC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>570</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>@2021</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
